src/java/org/apache/cassandra/service/opstate/KeyspaceTableOpStatePersister.java

```diff
@@ -124,6 +124,7 @@ private boolean writeStateToFile(File file, Map<KeyspaceTableKey, Instant> updat
         }
         catch (IOException e)
         {
+            log.warn("Failed to write state to file.", file.getAbsolutePath(), e);
```

Comment:
In this log and other instances of it, can we switch to:
```
log.warn("Failed to write state to file.", SafeArg.of("file", file.getName()), e);
```
to be able to get them in Loki